### ISSUE TITLE
Error when Using LearningWithNoiseLabels()

### ISSUE THREAD

**Issue by aretius** [cgnorthcutt/cleanlab#12]
Hey @cgnorthcutt ,

I am currently trying to improve my model's accuracy in classification. I know for sure that my labels have 20% noise in them. I am using Sklearn's Random Forest Classifier and my data is ~600K samples with 500 Features.

Environment Details :

- Mac
- Python 3.5.6
- Sklearn 0.21.3

Code: 
```
ln1=  LearningWithNoisyLabels(clf=RandomForestClassifier(class_weight='balanced'),seed = 2)
ln1.fit(X = x_train.values,s=y_train['prediction'].values)
```

Error Faced :

```
TypeError                                 Traceback (most recent call last)
<ipython-input-7-2636b16750dc> in <module>
      3 
      4 ln1=  LearningWithNoisyLabels(clf=RandomForestClassifier(class_weight='balanced'),seed = 2)
----> 5 ln1.fit(X = x_train.values,s=y_train['prediction'].values)

/anaconda3/envs/cleanlabel/lib/python3.5/site-packages/cleanlab/classification.py in fit(self, X, s, psx, thresholds, noise_matrix, inverse_noise_matrix)
    296             confident_joint = self.confident_joint,
    297             prune_method = self.prune_method,
--> 298             converge_latent_estimates = self.converge_latent_estimates,
    299         ) 
    300 

TypeError: get_noise_indices() got an unexpected keyword argument 'converge_latent_estimates
```

Exploring the code it seems the `function get_nosie_indices()` has no argument

**Comment by cgnorthcutt** [comment]
Thanks for using cleanlab. This error is due to an API change in the latest version. I'll have this fixed immediately. Give me a few minutes.

**Comment by cgnorthcutt** [comment]
Bug is fixed now (version 0.0.14) or you can pull the latest commit. Please confirm and I'll close this issue.

**Comment by aretius** [comment]
Yes the issues seems to be fixed now. Thanks for providing such a great library!
